common/src/test/java/org/apache/falcon/entity/parser/ProcessEntityParserTest.java

```diff
@@ -103,7 +103,7 @@ public void testParse() throws FalconException, JAXBException {
 
         Assert.assertEquals(process.getTags(),
                 "consumer=consumer@xyz.com, owner=producer@xyz.com, _department_type=forecasting");
-        Assert.assertEquals(process.getPipelines(), "testPipeline,dataReplication_Pipeline");
+        Assert.assertEquals(process.getPipelines(), "testPipeline");
```

Comment:
pipeline does not apply to a feed at all. Per process definition, a process can have multiple definitions. So, you should not make that assumption. Emit a metric per pipeline, if that is what is required.